posts/PvP-2-Robo-Proxy-Allin.md

```diff
@@ -5,12 +5,12 @@
 .. tags: pvp, early game, beginner, article, immortal, proxy
 .. category: videos
 .. link: https://www.youtube.com/watch?v=z3MFpm8l0O0
-.. description: <Put a short description here.>
+.. description: Simple and strong all-in
 .. type: text
 .. author: Harstem
 -->
 
-## Best Approach to PvP period
+## PvP-2-Robo-Proxy-Allin
```

Comment:
```suggestion

```

We don't actually need this title here because Nikola will use the title from the metadata at the start of the file.